### PR TITLE
v0.15.0

### PR DESCRIPTION
**Pull request by njtran**
**Description**

**How was this change tested?**

* Ran `make e2etests` with test infrastructure.

**Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by tzneal — DISMISSED** [review]
LGTM

File: website/content/en/preview/AWS/instance-types.md [hunk 1]
```diff
@@ -50,6 +51,7 @@ below are the resources available with some assumptions and after the instance o
  |kubernetes.io/arch|arm64|
  |kubernetes.io/os|linux|
  |node.kubernetes.io/instance-type|a1.large|
+ |topology.kubernetes.io/region|us-east-1|
```
Comment (bwagner5) [inline website/content/en/preview/AWS/instance-types.md:54]: we should remove

**Review by tzneal — APPROVED** [review]
lgtm

**Review by mbevc1 — APPROVED** [review]

**Comment by ellistarn** [comment]
https://github.com/mbevc1?achievement=yolo&tab=achievements

**Comment by mbevc1** [comment]
@ellistarn :joy:

**Pull request merged**